config/policies.js

```diff
@@ -37,7 +37,7 @@ module.exports.policies = {
   'game/get-list': 'isLoggedIn',
   'game/subscribe': ['isLoggedIn', 'hasGameId'],
   'game/spectate': ['isLoggedIn', 'hasGameId'],
-  'game/ready': 'isLoggedIn',
+  'game/ready': ['isLoggedIn', 'isInGame'],
```

Comment:
This checks that `req.session` has values for `usr` and `game`, which removes the need to check that in the ready action itself